src/clientAtom.ts

```diff
@@ -1,5 +1,5 @@
 import { cacheExchange, createClient, fetchExchange } from '@urql/core'
-import { atom } from 'jotai/vanilla'
+import { atom } from 'jotai'
```

Comment:
We should use `jotai/vanilla` which is the same thing, and allows to use `jotai-urql` without React.
```suggestion
import { atom } from 'jotai/vanilla'
```